src/App.tsx

```diff
@@ -1,14 +1,58 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+export function getFilteredTodo(
+  todos: Todo[],
+  filterType: string,
+  query: string,
+) {
+  const filterBy = todos.filter((todo) => {
+    switch (filterType) {
+      case 'all':
+        return todo;
+
+      case 'completed':
+        return todo.completed === true;
+
+      case 'active':
+        return todo.completed === false;
```

Comment:
you could just write this way. No need to compare with boolean.
```suggestion
      case 'completed':
        return todo.completed;

      case 'active':
        return !todo.completed;
```